### DISCUSSION TITLE
Customize Background Scanning

### DISCUSSION THREAD

**Discussion by ghost** [seemoo-lab/AirGuard#110] [Ideas]
Can you add an option to change how frequently the app scans for devices in the background?

**Comment by luke-jr** [comment]
Would be nice if it could be more passive too, and just detect the unsolicited beacons...

**Comment by Sn0wfreezeDev** [comment]
Unfortunately, Android puts in some hard limits here. So we could go higher than every 15min to 30min or once an hour, but not less than every 15min
